lib/timing.js

```diff
@@ -291,7 +291,7 @@ class Latency {
 			log.info(' 100%      %s ms (longest request)', this.maxLatencyMs);
 			log.info('');
 			Object.keys(results.errorCodes).forEach(errorCode => {
-				const padding = ' '.repeat(4 - errorCode.length);
+				const padding = ' '.repeat(errorCode.length < 4 ? 4 - errorCode.length : 1);
```

Comment:
I don't understand this bit?

Reply:
`errorCode` is modified to append the `customErrorCode`: (getRequestFinisherin [httpClient.js](https://github.com/alexfernandez/loadtest/blob/e38c1f6d6813ac28c68e562382d9f9ddae15960a/lib/httpClient.js#L207) )

```
					if (result.customErrorCode !== undefined) {
						errorCode = errorCode + ":" + result.customErrorCode
					}
```
Now it can be longer than 3, and get a 0 or minus parameter for `repeat`, which generates a runtime error.

This is a simple fix, may no be very beautiful, both in code and in the output. But it avoids the runtime error.